doc/vhpidirect/examples/quickstart.rst

```diff
@@ -42,3 +42,40 @@ When the required functionality is not available in pre-built libraries, custom
 This example shows how to bind custom C functions in VHDL as either procedures or functions. Four cases are included: ``custom_procedure``, ``custom_procedure_withargs``, ``custom_function`` and ``custom_function_withargs``. In all cases, the parameters are defined as integers, in order to keep it simple. See :ref:`COSIM:VHPIDIRECT:Declarations` for further details.
 
 Since either C sources or pre-compiled ``.o`` objects can be added, in C/C++ projects of moderate complexity, it might be desirable to merge all the C sources in a single object before elaborating the design.
+
+:cosimtree:`logicvectors <vhpidirect/quickstart/logicvectors>`
+**************************************************************
+
+Commonly signals in VHDL are of a logic type or a vector thereof (``std_logic`` and ``std_logic_vector``), coming from the ``ieee.std_logic_1164.*`` package.
+Because logic values are more than high and low (``1`` and ``0``), the VHDL values of logic types (``std_logic`` and ``std_ulogic``) are enumrations:
+
+0. 'U' 
+1. 'X' 
+2. '0' 
+3. '1' 
+4. 'Z' 
+5. 'W' 
+6. 'L' 
+7. 'H' 
+8. '-' 
+
+Because number of enumeration values is less than 256, logic values are transported in 8 bit words (a ``char`` type in C).
```

Comment:
Add ref to `Restrictions_on_foreign_declarations` in order to hold the assertion that are mapped as chars.

Reply:
Was so hesitant to do that... Here goes!

Reply:
`:ref:`COSIM:VHPIDIRECT:Declarations:Restrictions_on_foreign_declarations`` is just giving me errors

Reply:
`` :ref:`Restrictions_on_foreign_declarations` `` works.... but I see all of your refs are `COSIM:VHPIDIRECT:...`... that is what I meant by directory (esque) refs..

Reply:
The first syntax is taken from when the content was copied from the main docs, and it is preserved not to break the existing refs there. The second syntax is the one I am following, which matches the directory structure of the sources.

Reply:
I couldn't get a reference to the restrictions_on with the COSIM:VHPIDIRECT:... syntax. ooooohhhh... they're just different ways of labeling, the ref has to match the way of labeling... ok.